challenges/logic/todo/luka_jelacic/lib/blocs/todo_bloc.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:luka_jelacic/blocs/todo_event.dart';
+import 'package:luka_jelacic/blocs/todo_state.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:luka_jelacic/todoItem.dart';
+import 'package:uuid/uuid.dart';
+
+class TodoBloc extends Bloc<TodoEvent, TodoState> {
+  TodoBloc()
+      : super(TodoState(todoStatus: TodoStateStatus.InitialState, todos: []));
```

Comment:
- Preferirati [naming konvenciju](https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names) koja je preporucena od strane Dart tima, pojedine vrijednost enumeracije malim slovima.  
- Također ukoliko se radi o inicijalizaciji objekata sa više parametara kroz konstruktor, dodati zarez iza zadnjeg parametra zbog ljepšeg i preglednijeg formatiranja koda.